projects/autocomplete/src/lib/autocomplete.component.ts

```diff
@@ -243,7 +243,7 @@ export class AutocompleteComponent implements OnInit, OnChanges {
 
 @Directive({
   // tslint:disable-next-line
-  selector: '[autocomplete]'
+  selector: '[ng2-input-autocomplete]'
```

Comment:
could you keep [autocomplete] as well? I do not want to introduce breaking changes. '[autocomplete], [ng2-input-autocomplete]'

Reply:
This is just my opinion so you're free to take or leave it but I would say a breaking change is necessary because the autocomplete attribute introduces dangerous unintended side effects that people probably won't recognise straight away.

Reply:
If the autocomplete selector is left in it also won't fix the original issue.

Reply:
OK, I will introduce a major version. The projects using this library probably do not want to like breaking changes.

Reply:
cloud you change this name, 'ng2-input-autocomplete' to 'ng-autocomplete' (or something meaningful)? That name has been used as component.

Reply:
Changed to `input-autocomplete`